app/views/batches/index.html.erb

```diff
@@ -0,0 +1,65 @@
+<div class="row">
+  <div id="batch-create" class="bg-light col m-3 p-3">
+    <%= simple_form_for @batch do |f| %>
+      <%= f.error_notification message: f.object.errors[:base].to_sentence %>
+      <div class="row mb-3">
+        <div class="col">
+          <%= f.input :first_barcode %>
+        </div>
+        <div class="col">
+          <%= f.input :call_number %>
+        </div>
+      </div>
+      <div class="row mb-3">
+        <div class="col">
+          <%= f.input :start_box %>
+        </div>
+        <div class="col">
+          <%= f.input :end_box %>
+        </div>
+      </div>
+      <div class="row mb-3">
+        <div class="col">
+          <%= f.input :container_profile_uri, label: "Container profile", collection: @container_profiles, label_method: :name, value_method: :uri %>
+        </div>
+      </div>
+      <div class="row mb-3">
+        <div class="col">
+          <%= f.input :location_uri, label: "Location", collection: @locations, label_method: :title, value_method: :uri %>
+        </div>
+      </div>
+      <%= f.button :submit, class: 'btn-primary' %>
+    <% end %>
+  </div>
+</div>
+
+<h2>Unsynchronized</h2>
+<div id="unsynchronized-batches" class="bg-light col m-3 p-3">
+  <% (current_user&.batches&.order(created_at: :desc) || []).each do |batch| %>
```

Comment:
question: How are synchronized batches filtered out?

Reply:
It isn't right now, I figured we could handle that when synchronizing was a thing.